resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/circuitbreaker/monitoring/health/CircuitBreakersHealthIndicator.java

```diff
@@ -61,25 +61,33 @@ public CircuitBreakersHealthIndicator(CircuitBreakerRegistry circuitBreakerRegis
     public Health health() {
         Map<String, Health> healths = circuitBreakerRegistry.getAllCircuitBreakers().toJavaStream()
                 .filter(this::isRegisterHealthIndicator)
-                .collect(Collectors.toMap(CircuitBreaker::getName, CircuitBreakersHealthIndicator::mapBackendMonitorState));
+                .collect(Collectors.toMap(CircuitBreaker::getName, this::mapBackendMonitorState));
 
         return healthAggregator.aggregate(healths);
     }
 
     private boolean isRegisterHealthIndicator(CircuitBreaker circuitBreaker) {
         return circuitBreakerProperties.findCircuitBreakerProperties(circuitBreaker.getName())
                 .map(InstanceProperties::getRegisterHealthIndicator)
+                .orElse(true);
```

Comment:
We decided to disable HealthIndicator by default. Please set `false` the default value.

Reply:
I'm happy to do that but from what I understood from #607 they were disabled because they would cause the health status to go down.
Since that's no longer the case I wonder if we can turn them back on by default.

Thoughts?

Reply:
I think we should keep them disabled by default.

Reply:
I agree with @RobWin . That change may cause confusion to users.

Reply:
Ok, i'll make the change.